src/Types/PaymentType.res

```diff
@@ -885,10 +887,10 @@ let itemToCustomerObjMapper = customerDict => {
   (customerPaymentMethods, isGuestCustomer)
 }
 
-let createCustomerObjArr = dict => {
+let createCustomerObjArr = (dict, key) => {
```

Comment:
what is the need for key when it has a static value `"customerPaymentMethods"`

Reply:
It can also have value as - `savedPaymentMethods`